reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS.md

```diff
@@ -50,8 +50,8 @@ brew cask upgrade powershell
 
 See [About Brew](#about-brew) for information about Brew.
 
-Once you've installed Homebrew, installing PowerShell is easy.
-First, install [Cask-Versions][cask-versions] which lets you install alternative versions of cask packages:
+After you've installed Homebrew, you can install PowerShell.
+First, install [Cask-Versions that lets you install alternative versions of cask packages:
```

Comment:
The [Cask-Versions] hyperlink is broken. What happened?

Reply:
I deleted the link by mistake. Fixed the link and verified it works.

Reply:
Added code marks to the two instances of $PSVersionTable.